hazelcast/src/main/java/com/hazelcast/client/impl/ClientEngineImpl.java

```diff
@@ -149,20 +153,36 @@ private ClientExceptionFactory initClientExceptionFactory() {
         return new ClientExceptionFactory(jcacheAvailable);
     }
 
-    private Executor newExecutor() {
+    private Executor newClientExecutor() {
         final ExecutionService executionService = nodeEngine.getExecutionService();
         int coreSize = Runtime.getRuntime().availableProcessors();
 
         int threadCount = node.getProperties().getInteger(GroupProperty.CLIENT_ENGINE_THREAD_COUNT);
         if (threadCount <= 0) {
             threadCount = coreSize * THREADS_PER_CORE;
         }
+        logger.info("Creating new client executor with threadCount=" + threadCount);
```

Comment:
Hmm, really? It's just two lines on boot, not per query or sth. 
To me If I set the number of threads I want to be sure that the option is picked up and I don't have a typo or sth.
WDYT?